lib/dice.py

```diff
@@ -0,0 +1,24 @@
+from random import randint
+import re
+
+class Dice():
+
+    def split():
+        numbers = re.split('[d+]', roll)
+        return numbers
+
+    @static def roll(self, numbers):
+                for x in numbers:
+                    try:
+                        int(x)
```

Comment:
Instead of doing a try/except here, you can do a check first to see if it's safe
```
if x.is_alpha():
    x = int(x)
```

And instead of "swallowing" the exception by catching it, then doing a pile, you should probably let it fail, because it's an unrecoverable error

Reply:
I figured that if it did fail, we might want to have it feed back to the user the format that they were supposed to use, can we do that if it properly fails?

Reply:
That'll be up to whoever plumbs it into the command.

Reply:
"is_alpha" seems to check and make sure if everything in the string is alphabetical characters, would the suggested code not mean that if the code is in the correct format (all numbers) that this would return false? Am i missing something?